universal-application-tool-0.0.1/project/TailwindBuilder.scala

```diff
@@ -12,6 +12,14 @@ object TailwindBuilder {
         process = Option(
           Process("npx tailwindcss build -i ./app/assets/stylesheets/styles.css -o ./public/stylesheets/tailwind.css", base).run()
```

Comment:
Is this npx tailwinds still required here?

Reply:
The grunt processes only takes care of compiling on the changes. So either the npx tailwinds here or the dev one in the build.sbt is needed I'm guessing. I've mentioned that in another comment on this